instance/static/html/instance/index.html

```diff
@@ -24,6 +24,13 @@
       <li ng-repeat="instance in instanceList"
           ng-class="{active: instance == selected.instance}">
         <a ng-click="select('instance', instance)">{{ instance.name }}</a>
+        <div class="status-icon" ng-switch="instance.status">
+          <i class="fa fa-ellipsis-h" tooltip-placement="top" tooltip="Provisioning in progress" ng-switch-default></i>
+        </div>
```

Comment:
@chromano Yup, you're right about that case - see https://github.com/open-craft/opencraft/pull/13#discussion_r41695774 though for a way to still approach it the way I'm suggesting above.